app/models/ems_refresh.rb

```diff
@@ -204,6 +204,9 @@ def self.queue_merge(targets, ems, create_task = false)
   end
 
   def self.create_refresh_task(ems, targets)
+    targets = targets.collect do |t|
+      [t.first.demodulize, t.last]
```

Comment:
Can you change this to
```
targets.collect do |target_class, target_id|
```
So you don't need to do `t.first`/`t.last`?

Reply:
done